lib/screens/profile/profile_screen.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_hands/base/res/styles/app_styles.dart';
+
+class ProfileScreen extends StatelessWidget {
+  const ProfileScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Padding(
+        padding: const EdgeInsets.all(16.0),
+        child: Stack(  // Stack to overlay the edit button
+          children: [
+            Center( // Wrap the Column with the Center widget to center content
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.center, // Ensures horizontal centering of the children
+                children: <Widget>[
+                  const SizedBox(height: 20), // Space for the edit button
+                  const CircleAvatar(
+                    radius: 50,
+                    backgroundImage: AssetImage('assets/profile.jpg'), // Replace with your image path
```

Comment:
Consider referencing a variable here instead of hardcoding the string. Refence it from res/media.dart